filters/shedder/admission.go

```diff
@@ -456,7 +456,6 @@ func (ac *admissionControl) startSpan(ctx context.Context) (span opentracing.Spa
 	if parent != nil {
 		span = ac.tracer.StartSpan(admissionControlSpanName, opentracing.ChildOf(parent.Context()))
 		ext.Component.Set(span, "skipper")
-		ext.SpanKind.Set(span, "shedder")
```

Comment:
I am not sure if we really need non-standard kind here

Reply:
Can you check if we have streams that might select it and propose a change?

Reply:
Found one that also uses `operation IN ("admission_control")` which should be enough.

Reply:
Reached out to the owner and this was fixed.

Reply:
Can you add this to the release notes, too?
I don't know who will hit by that but that we do tell about.